packages/transaction-messages/src/transaction-message.ts

```diff
@@ -1,5 +1,9 @@
 import { AccountMeta, Instruction } from '@solana/instructions';
 
+/**
+ * @deprecated Use `TransactionMessage` instead.
+ * @todo In a future major version, remove the export
```

Comment:
I think the codebase already has `// TODO(#XYZ) In a future...` style comments where #XYZ is the GitHub issue number.

Also, I think it would be semver-esque to land this, do a release, then remove the word `export` with a `major` Changeset and do another release. They will have been done on the same day, but that still follows semver.

Thoughts?